lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -605,26 +619,15 @@ void LowerLayersPass::runOnModuleBody(FModuleOp moduleOp,
   });
 }
 
-/// Set an output file attribute on the provided operation if the pass has a
-/// non-empty test bench directory.
-void LowerLayersPass::maybeSetOutputDir(Operation *op) {
-  if (!testBenchDir.empty() && dut)
-    op->setAttr("output_file", hw::OutputFileAttr::getAsDirectory(
-                                   op->getContext(), testBenchDir,
-                                   /*excludeFromFileList=*/true));
+static Operation *setOutputFile(Operation *op, hw::OutputFileAttr file) {
+  op->setAttr("output_file", file);
+  return op;
 }
```

Comment:
In addition to the unused return, this is almost so simple now that it could be inlined. That said, the raw string in `setAttr` isn't ideal for inlining. Do we already have a function that does this?